StringStylizer/String+StringStylizer.swift

```diff
@@ -47,3 +47,11 @@ public extension String {
     }
 }
 
+
+public extension Optional where Wrapped == String {
+    
+    func stylize() -> StringStylizer<Styling> {
+        return (self ?? "").stylize()
+    }
+    
+}
```

Comment:
Thank you for your suggestion. I'd like to know the use case of this method. When a variable is optional, you can unwrap it as follows.

```swift
let str: String? = "foo"

str?.stylize() // => Optional<StringStylizer<Styling>>
```

Reply:
yeah, I was aware of this - but it gets a little messy when you're combining optional `NSAttributedString`'s with non optional ones via the `stylize()` api. I had defined this extension in a codebase I'm working on, and will show you a with vs. without comparison.

 I've really enjoyed writing code with `StringStylizer` in the form of
```swift
attributedString = (
 someString.stylizer().<stylecode>.attr +
someOtherString.stylizer().<stylecode>.attr
)
```
I think this approach reads really well.


for example - consider this situation.

```swift
//for some reason, this `Product` has strange optionality - as in, productName may be nil.
        titleLabel.attributedText = (
            "\(product.productQuantity)x ".stylize().font(.preferredFont(forTextStyle: .headline)).attr +
            product.productName.stylize().font(.preferredFont(forTextStyle: .callout)).attr
        )
```

and compare that to this.

```swift
        titleLabel.attributedText = (
            "\(product.productQuantity)x ".stylize().font(.preferredFont(forTextStyle: .headline)).attr +
                (product.productName?.stylize().font(.preferredFont(forTextStyle: .callout)).attr ?? NSAttributedString())
        )
```

I think the first option reads better as it is more communicating the intent, not as awkwardly dealing with optionals to please the compiler :)

But I am fine with this extension being more specific to my use-case - and I am happy to remove this change if you deem it unsuited for the rest of your Framework consumers.

Reply:
Thanks! That makes sense. I will merge this method to master branch. Can you make some unit tests in Test Target?

Reply:
Of course! I'll update some of the documentation (both in the Readme.md and in the code docs).

Reply:
@kazuhiro4949 , sorry for the long time to update. Please have a look through and let me know if you'd like me to change anything!